swap_meet/decor.py

```diff
@@ -1,2 +1,10 @@
-class Decor:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Decor(Item):
+    def __init__(self, condition = 0):
```

Comment:
Tiny style nitpick here and elsewhere: don't include spaces around the `=` for default values in a function definition. Instead, write it like `condition=0`